File: integrations/csharp/Oicana/ExportOnceResult.cs
namespace Oicana;

/// <summary>
/// Result of a one-shot template export.
/// </summary>
/// <param name="Document">The exported document.</param>
/// <param name="Warnings">Compilation warnings, or <c>null</c> if there were none.</param>
public record ExportOnceResult(Stream Document, string? Warnings);

File: integrations/java/oicana/src/main/java/com/oicana/ExportOnceResult.java
package com.oicana;

import java.util.Optional;

/**
 * Result of a one-shot template export.
 *
 * @param document the exported document
 * @param warnings compilation warnings, or an empty Optional if there were none
 */
public record ExportOnceResult(byte[] document, Optional<String> warnings) {
}
